ts/components/session/usingClosedConversationDetails.tsx

```diff
@@ -20,6 +20,10 @@ export function usingClosedConversationDetails(WrappedComponent: any) {
       void this.fetchClosedConversationDetails();
     }
 
+    public componentDidUpdate() {
```

Comment:
Might be better if we compare specific previous props to current props and only update if it has changed?

For example checking if 
```
isPublic,
        type,
        conversationType,
        isGroup,
        phoneNumber,
        id,
```
any of those change

Reply:
yes and no,
If a group gets updated by adding a group member, dropping an update like this will not trigger an update.

Reply:
Good point